package.json

```diff
@@ -7,8 +7,9 @@
     "dev": "vite dev",
     "build": "node scripts/pre-build.js && vite build",
     "preview": "vite preview",
-    "test": "npm run lint && npm run test:unit",
+    "test": "npm run lint && npm run check && npm run test:unit:coverage",
     "test:e2e": "playwright test",
+    "test:e2e:ui": "npm run test:e2e -- --ui",
```

Comment:
It's easy enough to just add the `-- --ui` flag but it's a couple less keystrokes, and I feel like I'll forget about this option otherwise 😅